### PR TITLE
Endpoints controller can print sensitive pod info when errors occur

### PR DESCRIPTION
**Pull request by smarterclayton**
In general, start to tone down the amount of log output.

### CODE REVIEW

File: pkg/service/endpoints_controller.go [hunk 1]
```diff
@@ -56,10 +56,10 @@ func (e *EndpointController) SyncServiceEndpoints() error {
 			continue
 		}
 
-		glog.V(3).Infof("About to update endpoints for service %v", service.Name)
+		glog.V(5).Infof("About to update endpoints for service %s/%s", service.Namespace, service.Name)
```
Comment (thockin) [inline pkg/service/endpoints_controller.go]: We should land on one syntax for printing namespace and name (for all sorts of objects) "namespace/name" vs "name.namespace" vs other.

namepsace/name has an advantage when both name and namespace are DNS subdomains: "foo.bar/quux.zorb" is clear, while "quux.zorb.foo.bar" is ambiguous.

If we all agree, we should start enforcing it in code reviews and fixing it up when we find it.

Comment (thockin) [inline pkg/service/endpoints_controller.go]: Maybe write a util.JoinName(namespace, name string) string function?

**Comment by thockin** [comment]
LGTM

File: pkg/service/endpoints_controller.go (hunk 1 above)
Comment (smarterclayton) [inline pkg/service/endpoints_controller.go]: Also UID and ResourceVersion

----- Original Message -----

Comment (smarterclayton) [inline pkg/service/endpoints_controller.go]: I'm thinking maybe meta.Identify(runtime.Object) or something.  We need to be able to crack the object efficiently and work for runtime.Unknown as well.

----- Original Message -----

**Pull request merged**